api/server/router/container/container_routes.go

```diff
@@ -736,6 +738,8 @@ func (s *containerRouter) wsContainersAttach(ctx context.Context, w http.Respons
 		if versions.GreaterThanOrEqualTo(version, "1.28") {
 			conn.PayloadType = websocket.BinaryFrame
 		}
+
+		// TODO: Close notifications
```

Comment:
In order to do this we need to switch to a lower-level websocket library such as https://pkg.go.dev/github.com/gobwas/ws